specification/cognitiveservices/AnomalyDetector/multivariate/models.tsp

```diff
@@ -72,33 +72,22 @@ model MultivariateBatchDetectionResultSummary {
   @doc("Error message when detection fails.")
   errors?: ErrorResponse[];
   @doc("Variable status.")
-  variableStates?: VariableState[];
+  variableStates?: MultivariateVariableState[];
 
   @doc("""
 Detection request for batch inference. This is an asynchronous inference that
 will need another API to get detection results.
 """)
-  setupInfo: MultivariateBatchDetectionOptions;
-}
-
-@doc("Error information that the API returned.")
-@error
-model ErrorResponse {
-
-  @doc("Error code.")
-  code: string;
-
-  @doc("Message that explains the error that the service reported.")
-  message: string;
+  setupInfo: MultivariateBatchDetectionSettings;
```

Comment:
This should be named `settings` or if we want a more verbose name `batchDetectionSettings`.

Reply:
Update this in projectedNames.

Reply:
@minkefusiji I dont see the projectedName pushed here